### PR TITLE
Fix swapping CVX token to USDC

### PR DESCRIPTION
**Pull request by nodimdim**

### CODE REVIEW

File: src/strategies/ConvexStrategy.sol [hunk 1]
```diff
@@ -241,17 +242,14 @@ contract ConvexStrategy is AccessStrategy {
         // Sell CVX rewards
         uint256 cvxBal = CVX.balanceOf(address(this));
         if (cvxBal > MIN_REWARD_AMOUNT) {
-            ISwapRouter.ExactInputSingleParams memory paramsCvx = ISwapRouter.ExactInputSingleParams({
-                tokenIn: address(CVX),
-                tokenOut: address(asset),
-                fee: 10_000,
+            ISwapRouter.ExactInputParams memory paramsCvx = ISwapRouter.ExactInputParams({
+                path: abi.encodePacked(address(CVX), fee, address(WETH), fee, address(asset)),
```
Comment (larryob) [inline src/strategies/ConvexStrategy.sol:246]: I'm pretty sure we want to be using the [0.05% pool](https://info.uniswap.org/#/pools/0x88e6a0c2ddd26feeb64f039a2c41296fcb3f5640) (500 fee).

File: .gas-snapshot [hunk 2]
```diff
@@ -111,14 +111,14 @@ L1AAVEStratTest:testTVL() (gas: 426307)
 L1BridgeEscrowTest:testclearFunds() (gas: 193033)
 L1BridgeEscrowTest:testclearFundsInvariants() (gas: 159376)
 L1BridgeEscrowTest:testclearFundsWithBadProof() (gas: 190575)
-L1DeltaNeutralTest:testClaimAndSellSushi() (gas: 1135395)
-L1DeltaNeutralTest:testCreatePosition() (gas: 1003827)
-L1DeltaNeutralTest:testDivest() (gas: 1318384)
-L1DeltaNeutralTest:testEndPosition() (gas: 1206346)
-L1DeltaNeutralTest:testOnlyAddressWithStrategistRoleCanEndPosition() (gas: 1038162)
+L1DeltaNeutralTest:testClaimAndSellSushi() (gas: 1129960)
```
Comment (larryob) [inline .gas-snapshot:114]: I don't think some of these gas costs should be changing. Can you run `foundryup`?

File: src/test/ConvexStrategy.t.sol [hunk 3]
```diff
@@ -155,7 +155,7 @@ contract ConvexStratTest is TestPlus {
 
         strategy.sellRewards(0, 0);
         uint256 crvUsdc = usdc.balanceOf(address(strategy));
-        assertApproxEqRel(crvUsdc, 10.3e6, 0.05e18);
+        assertApproxEqRel(crvUsdc, 10.3e6, 0.1e18);
 
         // CVX is about $5.92 as of block 16520958
         deal(address(strategy.CVX()), address(strategy), 10e18);
```
Comment (larryob) [inline src/test/ConvexStrategy.t.sol:161]: Let's try to sell about $1k worth of both crv and cvx.

**Review by larryob — CHANGES_REQUESTED** [review]
Looking good, the ETH/USDC pool should be changed.

File: src/test/ConvexStrategy.t.sol [hunk 4]
```diff
@@ -155,7 +155,7 @@ contract ConvexStratTest is TestPlus {
 
         strategy.sellRewards(0, 0);
         uint256 crvUsdc = usdc.balanceOf(address(strategy));
-        assertApproxEqRel(crvUsdc, 10.3e6, 0.05e18);
+        assertApproxEqRel(crvUsdc, 10.3e6, 0.1e18);
```
[the remark this answers is not in the document]
Comment (nodimdim) [reply src/test/ConvexStrategy.t.sol:158]: Turns out I was using the wrong price.

**Pull request merged**